playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+bottom of your body tag:
+
+```html
+<script src="//bit.ly/twilio-basic-v1"
+  sid="YOUR_TWILIO_ACCOUNT_SID_HERE"
+  token="YOUR_TWILIO_AUTH_TOKEN_HERE"
+></script>  
+```
+
+You can see this snippet of code inside the HTML tab of the JS Bin:
+
+> ![](img/html_tab.png)
+
+Before this will work though we need to change `"YOUR_TWILIO_ACCOUNT_SID_HERE"`
+and `"YOUR_TWILIO_AUTH_TOKEN_HERE"` to your actual Twilio sid and token.
+
+The facilitator should have given you an sid and a token. If you don't have one
+yet, ask your facilitator.
+
+If you are the facilitator (or you're just doing this workshop by yourself),
+follow the directions [here](../twilio/signup.md) to get your own Twilio
+sid and token.
+
+Once we have the sid and auth token, replace
+`YOUR_TWILIO_ACCOUNT_SID_HERE` and `YOUR_TWILIO_AUTH_TOKEN_HERE` with the actual
+Account SID and Auth Token.
+
+> ![](img/enter_account_sid.gif)
+
+### Writing the one line of code to send a text message
+
+Then type the below code into the left side of JS Bin _exactly_:
+
+```js
+Twilio.callAndSay("555-555-5555", "You just subscribed to Gossip Girl");
+```
+_**Don't forget the parentheses commas, and quotation marks.**_
+
+> ![](img/demo_1.gif)
+
+This code calls the phone number `555-555-5555` and says the message
+`Gossip Girl here, your one and only.` We don't want that.
+
+Change `555-555-5555` to your own phone number. Feel free to change the
+message too.
+
+> ![](img/demo_2.gif)
+
+Now go ahead and press the "Run" button.
+
+> ![](img/demo_3.gif)
+
+Your phone should ring shortly! Answer it!
+
+```
+If it doesn't work, ask a neighboring group to see if they can see what's wrong.
+Otherwise, raise your hand to ask a facilitator for help! We <3 questions!
+```
+
+### Having some fun
+
+If you want, you can now duplicate the code to call more people's phones! Ask
+your neighbors for their phone number!
+
+> ![](img/demo_4.gif)
+
+Go ahead and click run!
+
+![](img/celebrate.gif)
+
+## Outline for the rest of this tutorial
+
+### Part I
+
+This first section is linear and more guided.
+
+In this section, you will learn to build an HTML5 app that, when you press a
+a button, it will call a phone.
+
+[Here is a demo of the app we will
+make](http://jsbin.com/fawuda/86/edit?html,css,js,console,output). To make this
+work for you:
+
+1. make sure you change change `555-555-5555` to your phone number
+2. click the "Run with JS" button in the output and enter your Twilio sid and
+   token
+
+Here is an outline of what you'll learn on the way:
+
+1. [Send text messages and make phone calls with one line of code]
+   (#send-text-messages-and-make-phone-calls)
+2. [Creating an account on JS Bin](#saving-our-work)
+3. [Call a phone when a button is clicked]
+   (#call-a-phone-when-a-button-is-clicked)
+
+If you're more advanced, feel free to skip this section and go to the next.
+Do make sure that you create a JS Bin account first though.
```

Comment:
You can probably link to another doc here. Or recommend that they log in with github